src/live.rs

```diff
@@ -736,6 +736,10 @@ struct LiveIgnition {
 impl LiveIgnition {
     pub fn from_common(common: &CommonCustomizeConfig) -> Result<Self> {
         let mut conf = Self::default();
+        // Fail at runtime if the live OS coreos-installer doesn't support
```

Comment:
Not sure if it's worth it, though at least in the ISO case, we could e.g. extract `coreos-installer.service` and check for `/etc/coreos/installer.d` so that we fail at `coreos-installer iso customize` time.